Dockerfile

```diff
@@ -1,5 +1,5 @@
-FROM golang:1.17-alpine AS build
-RUN apk add --no-cache make build-base openssl-dev
+FROM registry.suse.com/bci/golang:1.18 AS build
```

Comment:
Most of Go code in other repos is already using Golan v1.18, so I think it's better to use this version.

Reply:
:+1: